packages/vuetify/src/components/VAlert/VAlert.tsx

```diff
@@ -60,6 +60,10 @@ export const makeVAlertProps = propsFactory({
     type: [Boolean, String, Function, Object] as PropType<false | IconValue>,
     default: null,
   },
+  iconSize: {
+    type: Number,
+    default: null,
+  },
```

Comment:
If we're going to add a new prop like this, we would need to do a few things:

- Switch target from `master` to `dev`
- Discuss where else this could be used if anywhere

Reply:
Other places usually respond to either straight or nested defaults `{ ... VIcon: { size: 20 } }` or Sass variables.

VAlert is overriding it because of `prominent` flag. I wish it would do either `zoom: 2` or `transform: scale(2)` in CSS instead. I am also not sure why it was bumped from 24 to 28 between v2 and v3.